src/main/java/io/lettuce/core/models/events/CommandStartedEvent.java

```diff
@@ -0,0 +1,29 @@
+package io.lettuce.core.models.events;
+
+import io.lettuce.core.protocol.RedisCommand;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author sokomishalov
+ */
+public class CommandStartedEvent<K, V, T> extends CommandBaseEvent<K, V, T> {
+
+    private final Instant startedAt;
+
+    public CommandStartedEvent(RedisCommand<K, V, T> command) {
+        super(command, new HashMap<>());
+        this.startedAt = Instant.now();
```

Comment:
To reduce allocations, let's keep this property as `long`

Reply:
fixed